### PR TITLE
fix: add missing styles

### PR DESCRIPTION
**Pull request by Jessica41**
Work of @scraly

### CODE REVIEW

**Comment by scraly** [comment]
thanks

**Comment by OVH-Team-Guides** [comment]
@scraly, could you please check that our edits are ok for you ?

File: pages/platform/public-cloud/how_to_use_terraform/guide.de-de.md [hunk 1]
```diff
@@ -600,7 +600,7 @@ If one of these business rules is not met, you will receive the following error:
 In that case, the only solution is to use the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.de/&ovhSubsidiary=de) to create a project.<br>
 You will then be challenged to validate that you are indeed the owner of the payment means used on your account (this challenge depends on the payment means and other parameters).
 
-Please understand these rules and extra human steps have been put in place as an extra safety for customers that migh have leaked their OVHcloud credentials.<br>
+Please understand these rules and extra human steps have been put in place as an extra safety for customers that mighthave leaked their OVHcloud credentials.<br>
```
Comment (scraly) [inline pages/platform/public-cloud/how_to_use_terraform/guide.de-de.md]: could have?

**Comment by scraly** [comment]
Hi,
I'm not fluent in english but seeing "mighthave" word seems weird for me.
Maybe replace to "could have" instead?

What do you think?

Thanks

**Comment by OVH-Team-Guides** [comment]
Thanks, that's a typo ;-)
I'll fix it

**Comment by scraly** [comment]
perfect :)

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @scraly

**Pull request merged**